llvm/utils/update_mc_test_checks.py

```diff
@@ -15,15 +15,14 @@
 
 mc_LIKE_TOOLS = [
     "llvm-mc",
-    "not llvm-mc",
 ]
```

Comment:
I would like to keep it since other update_** has similar code

Reply:
Then please add a comment similar to the one we have for llc:
```
# llc is the only llc-like in the LLVM tree but downstream forks can add
# additional ones here if they have them.
```

It would also be nice to fix the name to be all-capitals (probably with a separate patch).